docs/guide/http_cache.md

```diff
@@ -289,21 +291,66 @@ ezplatform:
 
 !!! note "Multiple Purge Servers"
 
-    If you need to set multiple purge servers, then you need to configure them in the YAML file.
+    If you need to set multiple purge servers, you must configure them in the YAML file.
+
+!!! note "Invalidating Varnish cache using tokens"
+
+    In setups where the Varnish server IP can change (for example on platform.sh/eZ Platform Cloud),
+    you can use token-based cache invalidation via [ez_purge_acl](https://github.com/ezsystems/ezplatform-http-cache/blob/2.0/docs/varnish/vcl/varnish5.vcl#L174).
+
+    In such a case use a strong, secure hash and make sure to keep the token secret.
+
+#### Ensure proper captcha behavior
+
+If your installation uses Varnish and you want users to be able to configure and use captcha in their forms, you must enable the sending of captcha data as a response to an Ajax request.
+Otherwise, Varnish prohibits the transfer of captcha data to the form, and users see an empty image.
+
+To enable the captcha over Ajax feature, modify the configuration file, for example `config/packages/ezplatform.yaml`, by adding the following code:
+
+```
+ezpatform:
+    system:
+        default:
+            form_builder:
+                captcha:
+                    use_ajax: true/false
+```
+
+!!! note
+
+    If you created a custom captcha block for your site by modifying the default file (`vendor/gregwar/captcha-bundle/Resources/views/captcha.html.twig`), you must make the following changes to the custom block template file:
```

Comment:
```suggestion
    If you created a custom Captcha block for your site by overriding the default file (`vendor/gregwar/captcha-bundle/Resources/views/captcha.html.twig`), you must make the following changes to the custom block template file:
```
modifying>overriding, I think that's clearer, because AFAIK the user's not supposed to modify files in `vendor`.